core/dbt/adapters/base/impl.py

```diff
@@ -1113,8 +1113,18 @@ def calculate_freshness(
         }
 
         # run the macro
+        # in older versions of dbt-core, the 'collect_freshness' macro returned the table of results directly
+        # starting in v1.5, by default, we return both the table and the adapter response (metadata about the query)
+        result: Union[
```

Comment:
Yes! Since `result` could be either:
- `AttrDict` containing `AdapterResponse` + `agate.Table` (v1.5+)

or

- just `agate.Table` (v1.4 and earlier &rarr; anyone with a custom override who is upgrading)